src_blog/_posts/New-Release-Grain-v0.4.0-Cereal.md

```diff
@@ -0,0 +1,67 @@
+---
+title: New Release! Grain v0.4.0 - Cereal
+subtitle: Announcing the release of Grain v0.4.0, Cereal, with NEAR support.
+date: 2021-09-07 12:00:00
+author: Blaine Bublitz
+cover: /blog/photos/cover/marie-bellando-mitjans-cBChXxQqEKM-unsplash.jpg
```

Comment:
This was like the first result I found, but I actually like it https://unsplash.com/photos/EV6C1LjH1Lk

Reply:
Lots of Grain yellow, I like it too

Reply:
Used that one! Thanks!